examples/custom-client-cxx/cli.cpp

```diff
@@ -26,14 +26,23 @@ static void print_status(aklite::cli::StatusCode ret) {
       std::cout << "SUCCESS: Unable to fetch updated TUF metadata, but stored metadata is valid";
       break;
     case aklite::cli::StatusCode::CheckinFailure:
-      std::cout << "FAILURE: Unable to get TUF metadata";
+      std::cout << "FAILURE: Error handling TUF metadata";
```

Comment:
Maybe `Failed to update` instead of `Error handling`, "handling" sounds a bit vague ?

Reply:
Adjusted.